frontend/encore/dev-server.rst

```diff
@@ -82,18 +76,14 @@ server SSL certificate:
           // ...
 
     +     .configureDevServerOptions(options => {
-    +         options.https = {
-    +             pfx: path.join(process.env.HOME, '.symfony/certs/default.p12'),
+    +         options.server = {
+    +             type: 'https',
+    +             options: {
+    +                 pfx: path.join(process.env.HOME, '.symfony/certs/default.p12'),
+    +             }
     +         }
     +     })
 
-
-.. caution::
-
-    Make sure to **not** pass the ``--https`` flag at the command line when
-    running ``encore dev-server``. This flag was required before 1.0, but now
-    will cause your config to be overridden.
```

Comment:
The first part of the note is now quite old as 1.0 was released about 2 years ago. For the second part (about overridden), I can't find anything in the code that would suggest that also passing `--https` would cause any type of overriding.